Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.py

```diff
@@ -0,0 +1,1304 @@
+        readable_response,
+        ['certificate_id', 'enabled', 'create_time', 'update_time', 'expiry_time'],
+        removeNull=True,
+        headerTransform=string_to_table_header)
+
+    command_results = CommandResults(
+        outputs_prefix='AzureKeyVault.Certificate',
+        outputs_key_field='id',
+        outputs=outputs,
+        raw_response=response,
+        readable_output=readable_output,
+        ignore_auto_extract=True
+    )
+
+    return command_results
+
+
+def get_certificate_policy_command(client: KeyVaultClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    List the policy for a certificate.
+
+    Args:
+        client (KeyVaultClient):  Azure Key Vault API client.
+        args (Dict[str, Any]): command arguments.
+
+    Returns:
+        CommandResults: Command results with raw response, outputs and readable outputs.
+
+    """
+    vault_name = args.get('vault_name')
+    certificate_name = args.get('certificate_name')
+    response = client.get_certificate_policy_request(
+        vault_name, certificate_name)
+    outputs = copy.deepcopy(response)
+    outputs['attributes'] = convert_time_attributes_to_iso(outputs['attributes'])
+    outputs['CertificateName'] = certificate_name
+
+    readable_output = tableToMarkdown(f'{certificate_name} Policy Information',
+                                      outputs,
+                                      ['id', 'key_props', 'secret_props', 'x509_props', 'issuer', 'attributes'],
+                                      removeNull=True, headerTransform=string_to_table_header)
+    command_results = CommandResults(
+        outputs_prefix='AzureKeyVault.CertificatePolicy',
+        outputs_key_field='id',
+        outputs=outputs,
+        raw_response=response,
+        readable_output=readable_output
+    )
+
+    return command_results
+
+
+def test_module(client: KeyVaultClient) -> None:
+    """
+     Test instance parameters validity.
+     Displays an appropriate message in case of invalid parameters.
+
+     Args:
+         client (KeyVaultClient):  Azure Key Vault API client.
+
+     Returns:
+         None
+     """
+    try:
+        client.ms_client.get_access_token(resource=MANAGEMENT_RESOURCE)
+        client.ms_client.get_access_token(resource=VAULT_RESOURCE)
+        client.list_key_vaults_request(1, 0)
+        return_results('ok')
+    except Exception as error:
+        if 'InvalidSubscriptionId' in str(error):
+            return_results('Invalid subscription ID. Please verify your subscription ID.')
+        elif 'SubscriptionNotFound' in str(error):
+            return_results('The given subscription ID could not be found.')
+        elif 'perform action' in str(error):
+            return_results('The client does not have Key Vault permissions to the given resource group name'
+                           ' or the resource group name does not exist.')
+
+
+def fetch_credentials(client: KeyVaultClient,
```

Comment:
I think I misinterpreted the fetch-credentials command implementation: I understood that I should implement the "Fetch all credentials" logic but not the "Fetch a specific set of credentials", since you said in the demo meeting that in this integration we don't need to fetch specific credentials from cache to the instance username and password. 
Clearly, it does not seem the case and I need to implement both. 

My question is - when we use "Fetch a specific set of credentials", the username and password should replace the parameters of Client ID and Client Secret in my instance?
Instead of: 
![image](https://user-images.githubusercontent.com/83762784/134587609-f2b791dd-d9ea-4cfb-bf93-106d82a40464.png)

I should use:
![image](https://user-images.githubusercontent.com/83762784/134587703-74b72604-9583-4e9d-830a-165611578796.png)

If I figured it in the wrong way, I think it will be for the best to make a short live session regards this issue.